sycl/test/extensions/properties/properties_kernel.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -fsyntax-only -Xclang -verify %s
```

Comment:
Alternatively you can remove the use of the deprecated aspects. We don't test other deprecated aspects here anyway.

Same goes for the other tests in this folder.